dkg/frost/frost.go

```diff
@@ -0,0 +1,756 @@
+package frost
+
+import (
+	"crypto/rand"
+	"encoding/json"
+	"fmt"
+
+	"github.com/bloxapp/ssv-spec/dkg"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/coinbase/kryptology/pkg/core/curves"
+	"github.com/coinbase/kryptology/pkg/dkg/frost"
+	"github.com/coinbase/kryptology/pkg/sharing"
+	ecies "github.com/ecies/go/v2"
+	"github.com/herumi/bls-eth-go-binary/bls"
+	"github.com/pkg/errors"
+)
+
+var thisCurve = curves.BLS12381G1()
+
+func init() {
+	_ = bls.Init(bls.BLS12_381)
+	_ = bls.SetETHmode(bls.EthModeDraft07)
+}
+
+type FROST struct {
+	identifier   dkg.RequestID
```

Comment:
better have some kind of a "state" we can get a root from so we can use it for the spect tests